src/index.js

```diff
@@ -239,16 +239,21 @@ exports.init = (Implementation) => {
       const collections = _.values(Schemas.schemas);
       configStore.integrator.defineCollections(collections);
 
-      // NOTICE: Check each Smart Action declaration to detect configuration errors.
-      _.each(collections, (collection) => {
-        if (collection.actions) {
-          _.each(collection.actions, (action) => {
-            if (action.fields && !_.isArray(action.fields)) {
+      collections
+        .filter((collection) => collection.actions)
```

Comment:
[ Spec? ] If I remember well, new projects generated by lumber v3 have an `actions` array by default, we should check the length to ensure it contains smart actions (better perf)